main.go

```diff
@@ -44,10 +44,6 @@ var (
 	// Insecure http cookies (only recommended for internal LANs/VPNs)
 	httpInsecure bool
 
-	// set based on httpAddr
-	httpIP   string
-	httpPort string
```

Comment:
There two were declared on line 136 of main.go using the go short declaration syntax of `:=` so these vars are being shadowed.